test/Common.FunctionalTests/Utilities/FunctionalTestsBase.cs

```diff
@@ -25,14 +26,7 @@ public FunctionalTestsBase(ITestOutputHelper output = null) : base(output)
             }
 
             // Currently hosting throws if the Servertype = IIS.
-            if (parameters.ServerType == ServerType.IIS)
-            {
-                _deployer = new IISDeployer(parameters, LoggerFactory);
-            }
-            else
-            {
-                _deployer = ApplicationDeployerFactory.Create(parameters, LoggerFactory);
-            }
+            _deployer = DeployerSelector.GetDeployer(parameters, LoggerFactory);
```

Comment:
Are we just silently ignoring parameters.Server?

Reply:
Yeah, we have to if we wanted minimal code change between the two projects. I can remove setting .Server.

Reply:
Hm, what about exposing current server type as a static?

Reply:
Sure, though I think we need to do something more. Is there a clean way to modify the test name to append the server type?

Reply:
Not that I know of.